manifests/apps/jupyter/deploy-python-model-quickstart.yaml

```diff
@@ -6,64 +6,64 @@ metadata:
     opendatahub.io/categories: 'Model serving,Getting started'
 spec:
   displayName: Deploying a sample Python application using Flask and OpenShift.
-  appName: jupyterhub
```

Comment:
Now that I'm reviewing this @MelissaFlinn @andrewballantyne @DaoDaoNoCode @shalberd  we as Red Hat, are no longer promoting s2i and we are moving away all our tech stack. 
Does it make sense to keep this tutorial? I would say that  we might need to deprecate it.

Reply:
I think we can keep it for now. We can always remove it as needed later.